code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm

```diff
@@ -118,10 +118,10 @@ Doesn't work on other aliens/AI.*/
 	set desc = "Secrete tough malleable resin."
 	set category = "Alien"
 
-	if(powerc(55))
+	if(powerc(55, TRUE))
 		var/choice = input("Choose what you wish to shape.","Resin building") as null|anything in list("resin wall","resin membrane","resin nest") //would do it through typesof but then the player choice would have the type path and we don't want the internal workings to be exposed ICly - Urist
 
-		if(!choice || !powerc(55))	return
+		if(!choice || !powerc(55, TRUE))	return
```

Comment:
would you make this follow indentation guidelines

Reply:
Done.